src/wallet/db.cpp

```diff
@@ -229,7 +229,7 @@ bool CDB::Recover(const std::string& filename, void *callbackDataIn, bool (*reco
 bool CDB::VerifyEnvironment(const std::string& walletFile, const fs::path& dataDir, std::string& errorStr)
 {
     LogPrintf("Using BerkeleyDB version %s\n", DbEnv::version(0, 0, 0));
```

Comment:
Not really related, but this gets logged by `-wallet` instance?
Maybe a better approach would be to do a single `"Using wallets [x,y,z]\n"` would be better?